spec/js/views/app_view_spec.js

```diff
@@ -1,7 +1,8 @@
 define([
```

Comment:
It would have been nice to see a test for the `initializer` asserting the todo view was set